src/components/LoansByCategory/HelpmeChoose/HelpmeChooseRecommendations.vue

```diff
@@ -1,4 +1,22 @@
 <template>
-	<div>
+	<div class="tw-flex">
+		<!-- eslint-disable max-len -->
+		<helpme-choose-borrower-selector
+			image-url="https://res.cloudinary.com/kiva/w_960,h_600,c_fill,g_faces/a_ignore,fl_progressive,q_auto:best,f_auto/remote/ea3be8ae591d38de82c7a81b3657b22e.jpg"
```

Comment:
Is this a static URL that we can guarantee won't change?

Reply:
This will be dynamic, static for testing the component.